### PR TITLE
Fabo/fix page block

### PR DESCRIPTION
**Pull request by faboweb**
**Description:**

The blocks module was cluttered with outdated properties. The page blocks was flick flacking between blocks. The moniker in the transactions were not showing properly.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/filters/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const formatBech32 = (address, longForm = false, length = 4) => {
 
 export const resolveValidatorName = (address, validators) => {
   if (validators[address]) {
-    return validators[address].moniker
+    return validators[address].description.moniker
```
Comment (colw) [inline src/filters/index.js]: The validator object is a flat structure from GraphQL now.

You can see the TransactionItems won't fill out properly.

File: src/scripts/transaction-utils.js [hunk 2]
```diff
@@ -16,13 +16,13 @@ const getFees = (transaction, defaultDenom = "ATOM") => {
   }
 }
 
-const makeTxObject = (x, fees, memo, time, height) => {
+const makeTxObject = (tx, fees, memo, time, height, hash) => {
   return {
-    ...x,
-    key: `${x.type}_${time}_${JSON.stringify(x.value)}`,
+    ...tx,
+    key: hash,
```
Comment (colw) [inline src/scripts/transaction-utils.js]: My bad, I left this in. I wonder if we get a speed boost :)

File: src/vuex/modules/transactions.js [hunk 3]
```diff
@@ -133,6 +133,8 @@ export default ({ node }) => {
       const enrichedTransactions = await Promise.all(
         transactions.map(async tx => {
           const blockMetaInfo = await dispatch(`queryBlockInfo`, tx.height)
+          // if (!blockMetaInfo) return tx
```
Comment (colw) [inline src/vuex/modules/transactions.js]: Comment. Need to keep?

File: tests/unit/specs/filters/filters.spec.js [hunk 4]
```diff
@@ -52,7 +52,9 @@ describe(`resolveValidatorName Filter`, () => {
   beforeEach(() => {
     validators = {
       cosmosvaloper1xyz: {
-        moniker: "Big Val"
+        description: {
```
Comment (colw) [inline tests/unit/specs/filters/filters.spec.js]: As above, this needs reverting.

**Comment by faboweb** [comment]
@colw removing the description fix:

image

**Comment by faboweb** [comment]
Got it, the validators come from gql now and not from the store

**Pull request merged**